pkg/config/read.go

```diff
@@ -69,15 +71,11 @@ func ToEnv(cfg Config) ([]string, error) {
 
 // it's a mapping of how config env option should be transliterated to the elemental CLI
 var defaultOverrides = map[string]string{
-	"ELEMENTAL_INSTALL_CONFIG_URL":       "ELEMENTAL_INSTALL_CLOUD_INIT",
-	"ELEMENTAL_INSTALL_ISO_URL":          "ELEMENTAL_INSTALL_ISO",
-	"ELEMENTAL_INSTALL_POWER_OFF":        "ELEMENTAL_POWEROFF",
-	"ELEMENTAL_INSTALL_DEVICE":           "ELEMENTAL_INSTALL_TARGET",
-	"ELEMENTAL_INSTALL_CONTAINER_IMAGE":  "ELEMENTAL_INSTALL_SYSTEM",
-	"ELEMENTAL_INSTALL_NO_FORMAT":        "ELEMENTAL_INSTALL_NO_FORMAT",
-	"ELEMENTAL_INSTALL_TTY":              "ELEMENTAL_INSTALL_TTY",
-	"ELEMENTAL_INSTALL_DEBUG":            "ELEMENTAL_DEBUG",
-	"ELEMENTAL_INSTALL_REGISTRATION_URL": "ELEMENTAL_REGISTRATION_URL",
+	"ELEMENTAL_INSTALL_CONFIG_URL": "ELEMENTAL_INSTALL_CLOUD_INIT",
```

Comment:
isnt there a way we can drop this already? It hurts my heart everytime I see this, it should not eb needed, we control both projects :crying_cat_face: